cmd/machine-controller/main.go

```diff
@@ -166,7 +166,7 @@ func main() {
 	flag.Var(&nodeContainerdRegistryMirrors, "node-containerd-registry-mirrors", "Configure registry mirrors endpoints. Can be used multiple times to specify multiple mirrors")
 	flag.StringVar(&caBundleFile, "ca-bundle", "", "path to a file containing all PEM-encoded CA certificates (will be used instead of the host's certificates if set)")
 	flag.BoolVar(&nodeCSRApprover, "node-csr-approver", true, "Enable NodeCSRApprover controller to automatically approve node serving certificate requests")
-	flag.StringVar(&podCidr, "pod-cidr", "172.25.0.0/16", "The network ranges from which POD networks are allocated")
+	flag.StringVar(&podCIDRs, "pod-cidrs", "172.25.0.0/16", "The network ranges from which POD networks are allocated")
```

Comment:
This is breaking change to flags, is it _really_ justified?

Reply:
The way around is to have both, `-pod-cidr` and `-pod-cidrs` flags, and then in runtime merge their values if anything been provided.

Reply:
OK, you're right, nether kubermatic nor kubeone uses it, so it's OK from my PoV.

Reply:
👍

we can also pass same flag multiple times to get multiple values if it breaks anything at all